plugins/obs-ffmpeg/obs-ffmpeg-source.c

```diff
@@ -310,9 +320,12 @@ static void ffmpeg_source_update(void *data, obs_data_t *settings)
 
 		obs_source_set_async_unbuffered(s->source, false);
 	}
+	input_options = (char *)obs_data_get_string(settings,
+		"input_options");
```

Comment:
Style:  Line breaks occur after 80 columns.   Lines 323-324 can be a single line.

Reply:
done, thanks a lot !